backend/Pipfile

```diff
@@ -50,6 +50,7 @@ urlextract = "*"
 jsonref = "*"
 unittest-xml-reporting = "*"
 tblib = "*"
+colorama = "*"
```

Comment:
Do we need this dependency? I just delete this from the `Pipfile` unless there is something we need it for (don't want to download more dependencies than necessary).

Reply:
This was just because `black .` wasn't running before, but I've removed it. Other than that I've implemented the other changes, although every time I run pipenv install it'll change one line in my Pipfile.lock, I'm not sure if that's a problem?